dapp/src/utils/hooks.js

```diff
@@ -145,3 +147,18 @@ export function usePrevious(value) {
   })
   return ref.current
 }
+
+export function useOverrideAccount() {
+  const router = useRouter()
+  const isValid =
```

Comment:
Going by this logic, `override_account` can have a value but it won't be always valid. But I don't see the `isValid` being used anywhere before using the `overrideAccount`.

Perhaps, make sure the function doesn't return `overrideAccount` if `isValid` is false

Reply:
` const overrideAccount =
    pathname === '/history' && override_account && isValid
      ? override_account
      : null`

There is this logic below so only it will only return `override_account` if `isValid` is true else it will return null

Reply:
Ah, yes. I missed it